tests/conftest.py

```diff
@@ -62,6 +62,12 @@ def add_reference_response_to_mock(mock: Mock, url: str) -> Mock:
         mock.json.return_value = json.load(
             open("./tests/json_data/serviceChannelTypes.json")
         )
+    elif (
+        "reference-data/adms/publisher-types" in url
+    ):  # Hva blir linken her? 🤔 https://data.norge.no/reference-data/adms/publisher-types
```

Comment:
Stemmer! Kan testes i terminal med å kjøre `curl -H "Accept: application/json" https://data.norge.no/reference-data/adms/publisher-types` (og evt. pipe til ` | jq` for prettyprint).

Reply:
Men her får jeg noe mock-feil 

```
FAILED tests/test_public_service.py::test_complete_public_services - TypeError: 'Mock' object is not iterable
FAILED tests/test_public_service.py::test_parse_multiple_public_services - TypeError: 'Mock' object is not iterable
FAILED tests/test_public_service.py::test_parse_cpsvno_services - TypeError: 'Mock' object is not iterable
```